config/acme/machines/template.case.run

```diff
@@ -32,21 +32,21 @@ import argparse, doctest
 def parse_command_line(args, description):
 ###############################################################################
     parser = argparse.ArgumentParser(
-        usage="""\n%s [--verbose]
+        usage="""\n{} [--verbose]
 OR
-%s --help
+{} --help
 OR
-%s --test
+{} --test
 
 \033[1mEXAMPLES:\033[0m
     \033[1;32m# case.run SMS\033[0m
-    > %s
-""" % ((os.path.basename(args[0]), ) * 4),
+    > {}
+""".format(((os.path.basename(args[0]), ) * 4),
```

Comment:
Will fix. In this case it would be `{0}` since lists/arrays start at 0 in python.